web/src/excluded-aggregated-exchanges.json

```diff
@@ -1,11 +1,37 @@
 {
-  "exchangesToExclude": [
+  "exchangesToExcludeCountryView": [
+    "AT->IT-NO",
+    "AX->SE-SE3",
```

Comment:
We need to add an `AX->SE` exchange in the `exchanges.json` file (couldn't add a comment there).

Reply:
Done. Will push with next commit